client/src/components/new-request/analyze-results.vue

```diff
@@ -244,8 +247,36 @@ export default class AnalyzeResults extends Vue {
     }
   }
 
+  get allDesignationsStripped () {
+    return this.stripAllDesignations(this.originalName)
+  }
+  get allowProceed () {
+    /* newReqModule.designationIssueTypes is a list of all the designation-related issue types.  designationIsFixed
+     will not be true unless one of these issues was solved.  some sets of issues will not include a designation-related
+     issue and this will need to return true in these cases even tho designationIsFixed will be false */
+    if (!this.hasDesignationIssue) {
+      if (this.isLastIndex) {
+        return true
+      }
+    }
```

Comment:
Might be able to simplify this condition into one statement. 
ie `(!this.hasDesignationIssue && this.isLastIndex)`

Reply:
yup